modules/doc/content/help/faq/conda_issues.md

```diff
@@ -30,10 +30,10 @@ Conda issues can be the root cause for just about any issue on this page. Scroll
 
 - #### command not found: conda
 
-  You have yet to install conda, or your path to it is incorrect or not set. You will need to recall how you installed conda. Our instructions ask to have Miniconda3 installed to your home directory: `~/miniconda3`. Which requires you to set your PATH accordingly:
+  You have yet to install conda, or your path to it is incorrect or not set. You will need to recall how you installed conda. Our instructions ask to have Miniconda3 installed to your home directory: `~/mambaforge3`. Which requires you to set your PATH accordingly:
```

Comment:
Do you want to change Miniconda3 in the line of text (middle of second wrapped line on my screen) to mambaforge3?

Reply:
Good call - I swore I changed that but perhaps I didn't save before staging.